xbmc/guilib/guiinfo/MusicGUIInfo.cpp

```diff
@@ -99,16 +100,25 @@ bool CMusicGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
         return true;
       case PLAYER_TITLE:
         value = tag->GetTitle();
+        if (!tag->GetStationName().empty() && tag->GetTitle().empty())
+        {
+          value.clear();
```

Comment:
`value` is equal to `tag.GetTitle()`at this point. So, if `tag->GetTitle().empty()`must be true (&&) to get here, `value` is already empty. This line can be removed.

Reply:
I could remove the `tag->GetTitle().empty()` bit but the rest of it has to remain otherwise it will cascade down and end up displaying a title from the path which would not be correct.

Reply:
For my PR I had to do something similar to avoid the PVR filename from appearing https://github.com/xbmc/xbmc/pull/15695#issuecomment-595958311

Reply:
It is `value.clear()` that can be removed